container/root/worker.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Entrypoint for utilizing as a worker pool instead of a web server
+# Based on configuration, can run multiple instances of a single worker process
+
+# TODO: this is straight copy-paste from docker-nginx's run.sh, needs to be split into a separate file than the nginx-start
+
+RUN_SCRIPTS=/run.d
+STATUS=0
+SUPERVISOR_CONF=/etc/supervisor/conf.d/worker.conf
+
+export CONTAINER_ROLE='worker'
+
+
+# Run shell scripts (ending in .sh) in run.d directory
+
+# When .sh run scripts fail (exit non-zero), container run will fail
+# NOTE: if a .sh script exits with 99, this is our stop signal, container will exit cleanly
+
+for file in $RUN_SCRIPTS/*.sh; do
+
+  echo "[run.d] executing ${file}"
+
+  /bin/bash $file
+
+  STATUS=$?  # Captures exit code from script that was run
+
+  if [[ $STATUS == $SIGNAL_BUILD_STOP ]]
+  then
+    echo "[run.d] exit signalled - ${file}"
+    exit # Exit cleanly
+  fi
+
+  if [[ $STATUS != 0 ]]
+  then
+    echo "[run.d] failed executing - ${file}"
+    exit $STATUS
+  fi
+
+done
+
+
+WORKER_RUNNER=$1
+WORKER_QUANTITY=$2
+WORKER_NAME=$3
+WORKER_PARAM_1=$4
+WORKER_PARAM_2=$5
```

Comment:
Any reason to have 2 WORKER_PARAMS? Wouldn't a single  space separated string do the trick?